test/integration/targets/pip/vars/main.yml

```diff
@@ -2,6 +2,12 @@ pip_test_package: sampleproject
 pip_test_packages:
   - sampleproject
   - decorator
+pip_test_pkg_ver:
```

Comment:
I would also check installs from Git + `--editable` installs.

Reply:
There is already one: https://github.com/ansible/ansible/blob/devel/test/integration/targets/pip/tasks/pip.yml#L145